pkg/userdata/rhel/provider.go

```diff
@@ -176,20 +176,20 @@ write_files:
     sysctl --system
 
 {{- /* Make sure we always disable swap - Otherwise the kubelet won't start */}}
-    cp /etc/fstab /etc/fstab.orig
-    cat /etc/fstab.orig | awk '$3 ~ /^swap$/ && $1 !~ /^#/ {$0="# commented out by cloudinit\n#"$0} 1' > /etc/fstab.noswap
-    mv /etc/fstab.noswap /etc/fstab
+    sed -i.orig '/.*swap.*/d' /etc/fstab
     swapoff -a
     {{ if ne .CloudProviderName "aws" }}
 {{- /*  The normal way of setting it via cloud-init is broken, see */}}
 {{- /*  https://bugs.launchpad.net/cloud-init/+bug/1662542 */}}
     hostnamectl set-hostname {{ .MachineSpec.Name }}
     {{ end }}
 
-    dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+    yum install -y yum-utils
+    yum-config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+    yum-config-manager --save --setopt=docker-ce-stable.module_hotfixes=true
```

Comment:
I think that a comment on this line explaining why this is needed would be useful.
BTW I think this is required for RHEL8 only.

Reply:
It's required on CentOS8 too. And on CentOS7 it's a noop.

Reply:
The problem is, we can test RHEL whatsoever, due to lack of subscriptions. Our subscription has expired and it hasn't been updated for RHEL

Reply:
This command was tested manually on live system multiple times

Reply:
my point was simply that we should have a comment on this line to explain the reason why we are doing this as it is not self evident

Reply:
indeed, added the comment 👍